Makefile

```diff
@@ -762,6 +762,15 @@ test-azure-ipam: ## run the unit test for azure-ipam
 kind:
 	kind create cluster --config ./test/kind/kind.yaml
 
+test-k8se2e: test-k8se2e-build test-k8se2e-only ## Alias to run build and test
+
+test-k8se2e-build: ## Build k8s e2e test suite
+	cd hack/scripts && bash ./k8se2e.sh $(GROUP) $(CLUSTER)
```

Comment:
Will this work? Given we don't have variables `Group` or `Cluster` in the file ?

Reply:
Yes, these scripts were heavily reliant on those arguments being present in command to begin with. Having a default did not provide any additional benefit.